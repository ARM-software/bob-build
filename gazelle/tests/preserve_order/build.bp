// Main build file

bob_filegroup {
    name: "fg_101",
    srcs: ["srcs/input_101.c"],
}

bob_filegroup {
    name: "fg_102",
    srcs: ["srcs/input_102.c"],
}

bob_glob {
    name: "glob_102_1",
    srcs: ["lib/**/*_102_1.c"],
    allow_empty: true,
}

bob_filegroup {
    name: "fg_103",
    srcs: ["srcs/input_103.c"],
}

bob_filegroup {
    name: "fg_104",
    srcs: ["srcs/input_104.c"],
}

bob_filegroup {
    name: "fg_105",
    srcs: ["srcs/input_105.c"],
}

bob_filegroup {
    name: "fg_106",
    srcs: ["srcs/input_106.c"],
}

bob_glob {
    name: "glob_lib_107",
    srcs: ["lib/**/*_107.c"],
    exclude: ["lib/**/exclude_*_107.cpp"],
    exclude_directories: true,
    allow_empty: true,
}

bob_glob {
    name: "glob_lib_108",
    srcs: ["lib/**/*_108.c"],
    exclude: ["lib/**/exclude_*_108.cpp"],
    exclude_directories: true,
    allow_empty: true,
}

bob_glob {
    name: "glob_lib_109",
    srcs: ["lib/**/*_109.c"],
    exclude: ["lib/**/exclude_*_109.cpp"],
    exclude_directories: true,
    allow_empty: true,
}

bob_glob {
    name: "glob_lib_110",
    srcs: ["lib/**/*_110.c"],
    exclude: ["lib/**/exclude_*_110.cpp"],
    exclude_directories: true,
    allow_empty: true,
}

bob_glob {
    name: "glob_lib_111",
    srcs: ["lib/**/*_111.c"],
    exclude: ["lib/**/exclude_*_111.cpp"],
    exclude_directories: true,
    allow_empty: true,
}

bob_glob {
    name: "glob_lib_112",
    srcs: ["lib/**/*_112.c"],
    exclude: ["lib/**/exclude_*_112.cpp"],
    exclude_directories: true,
    allow_empty: true,
}
