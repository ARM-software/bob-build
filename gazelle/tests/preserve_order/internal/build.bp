// internal build.bp

bob_filegroup {
    name: "fg_201",
    srcs: ["srcs/input_201.c"],
}

bob_filegroup {
    name: "fg_202",
    srcs: ["srcs/input_202.c"],
}

bob_filegroup {
    name: "fg_203",
    srcs: ["srcs/input_203.c"],
}

bob_filegroup {
    name: "fg_204",
    srcs: ["srcs/input_204.c"],
}

bob_glob {
    name: "glob_204_1",
    srcs: ["lib/**/*_204_1.c"],
    allow_empty: true,
}

bob_filegroup {
    name: "fg_205",
    srcs: ["srcs/input_205.c"],
}

bob_filegroup {
    name: "fg_206",
    srcs: ["srcs/input_206.c"],
}

bob_glob {
    name: "glob_lib_207",
    srcs: ["lib/**/*_207.c"],
    exclude: ["lib/**/exclude_*_207.cpp"],
    exclude_directories: true,
    allow_empty: true,
}

bob_glob {
    name: "glob_lib_208",
    srcs: ["lib/**/*_208.c"],
    exclude: ["lib/**/exclude_*_208.cpp"],
    exclude_directories: true,
    allow_empty: true,
}

bob_glob {
    name: "glob_lib_209",
    srcs: ["lib/**/*_209.c"],
    exclude: ["lib/**/exclude_*_209.cpp"],
    exclude_directories: true,
    allow_empty: true,
}

bob_glob {
    name: "glob_lib_210",
    srcs: ["lib/**/*_210.c"],
    exclude: ["lib/**/exclude_*_210.cpp"],
    exclude_directories: true,
    allow_empty: true,
}

bob_glob {
    name: "glob_lib_211",
    srcs: ["lib/**/*_211.c"],
    exclude: ["lib/**/exclude_*_211.cpp"],
    exclude_directories: true,
    allow_empty: true,
}

bob_glob {
    name: "glob_lib_212",
    srcs: ["lib/**/*_212.c"],
    exclude: ["lib/**/exclude_*_212.cpp"],
    exclude_directories: true,
    allow_empty: true,
}
