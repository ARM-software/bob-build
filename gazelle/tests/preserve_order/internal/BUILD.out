load("@bazel_skylib//rules:common_settings.bzl", "bool_flag")

bool_flag(
    name = "feature_b",
    build_setting_default = True,
)

config_setting(
    name = "config_feature_b",
    flag_values = {
        ":feature_b": "true",
    },
)

filegroup(
    name = "fg_201",
    srcs = [
        "srcs/input_201.c",
    ],
)

filegroup(
    name = "fg_202",
    srcs = [
        "srcs/input_202.c",
    ],
)

filegroup(
    name = "fg_203",
    srcs = [
        "srcs/input_203.c",
    ],
)

filegroup(
    name = "fg_204",
    srcs = [
        "srcs/input_204.c",
    ],
)

filegroup(
    name = "glob_204_1",
    srcs = glob(
        ["lib/**/*_204_1.c"],
        exclude_directories = 1,
        allow_empty = True,
    ),
)

filegroup(
    name = "fg_205",
    srcs = [
        "srcs/input_205.c",
    ],
)

filegroup(
    name = "fg_206",
    srcs = [
        "srcs/input_206.c",
    ],
)

filegroup(
    name = "glob_lib_207",
    srcs = glob(
        ["lib/**/*_207.c"],
        exclude = ["lib/**/exclude_*_207.cpp"],
        exclude_directories = 1,
        allow_empty = True,
    ),
)

filegroup(
    name = "glob_lib_208",
    srcs = glob(
        ["lib/**/*_208.c"],
        exclude = ["lib/**/exclude_*_208.cpp"],
        exclude_directories = 1,
        allow_empty = True,
    ),
)

filegroup(
    name = "glob_lib_209",
    srcs = glob(
        ["lib/**/*_209.c"],
        exclude = ["lib/**/exclude_*_209.cpp"],
        exclude_directories = 1,
        allow_empty = True,
    ),
)

filegroup(
    name = "glob_lib_210",
    srcs = glob(
        ["lib/**/*_210.c"],
        exclude = ["lib/**/exclude_*_210.cpp"],
        exclude_directories = 1,
        allow_empty = True,
    ),
)

filegroup(
    name = "glob_lib_211",
    srcs = glob(
        ["lib/**/*_211.c"],
        exclude = ["lib/**/exclude_*_211.cpp"],
        exclude_directories = 1,
        allow_empty = True,
    ),
)

filegroup(
    name = "glob_lib_212",
    srcs = glob(
        ["lib/**/*_212.c"],
        exclude = ["lib/**/exclude_*_212.cpp"],
        exclude_directories = 1,
        allow_empty = True,
    ),
)
